client/src/forms/schema.js

```diff
@@ -0,0 +1,30 @@
+import SimpleSchema from 'simpl-schema';
+import SimpleSchema2Bridge from 'uniforms-bridge-simple-schema-2';
+import { LongTextField } from 'uniforms-ionic';
+
+const s = new SimpleSchema({
+  title: {
```

Comment:
Since we use mongo we can add some of the elements now to the task to present capabilities of the forms. We use here an only string. Maybe some checkboxes etc. 

Suggestions:
- suggestedStartTime
- suggestedStartDay
- finishedNote (when task is completed it is marked with note)
- incidents (Long string field to mark some problems with task)
- priority 
- links (string with possible external references)

Reply:
This is optional, but we can copy some stuff from https://openvolunteer.org

I will add relationship later on as with relationship this will be much better (something like task having comments rather than finishedNote field

Reply:
Yeah that sounds like a good idea. I can flesh this out a bit and add a few more fields to this

Reply:
Thinking about this a little bit, maybe this part we can think about add it in as a separate PR. Just so we can get the forms added in without too much of a drastic change (from the UI side).

Reply:
Yes.. I'm happoy to add this. I will create issue and assign this to myself.